app/views/administrate/application/_sidebar.html.erb

```diff
@@ -11,7 +11,7 @@ as defined by the DashboardManifest.
   <% DashboardManifest::DASHBOARDS.each do |resource| %>
     <li>
       <%= link_to(
-        resource.to_s.titleize,
+        display_resource_name(resource),
```

Comment:
@mcmire completely agreed. I'm planning to change the `DashboardManifest::DASHBOARDS` constant to an array of constants instead of an array of symbols. This would let us support namespaced models (#32, #179), and hopefully get rid of a lot of name inferences.